letsdrill.rb

```diff
@@ -1,13 +1,24 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ grade = ""
+  grade_ranges = {
+  	"F" => 60,
+  	"D" => 70,
+  	"C" => 80,
+  	"B" => 90
+  }
 
+  grade_ranges.each do |letter, upper_bound|
+  	if (integer < upper_bound)
+  		grade = letter
+  		return grade
+  	else
+  		grade = "A"
+  	end
+  end
```

Comment:
This is better.  Do you need that initial empty string?  Also, you can clean up that each-block a little: think about why the `return` statement is needed in the `if` section, but not in the `else`.  And where the assignment to `grade` gets used if the grade is an A.